From 8b0b7e31cb2f8f1e31eb75f1d33c8decef0cd7c9 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Mon, 6 Aug 2018 22:32:42 +0200
Subject: [PATCH 1/1] efi_loader: efi_dp_get_next_instance() superfluous
 statemt

Remove a superfluous statement in efi_dp_get_next_instance().

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 lib/efi_loader/efi_device_path.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/lib/efi_loader/efi_device_path.c b/lib/efi_loader/efi_device_path.c
index 6040bcff19..172317b9a0 100644
--- a/lib/efi_loader/efi_device_path.c
+++ b/lib/efi_loader/efi_device_path.c
@@ -387,7 +387,6 @@ struct efi_device_path *efi_dp_get_next_instance(struct efi_device_path **dp,
 		*size = 0;
 	if (!dp || !*dp)
 		return NULL;
-	p = *dp;
 	sz = efi_dp_instance_size(*dp);
 	p = dp_alloc(sz + sizeof(END));
 	if (!p)
-- 
2.18.0

